internal/receiver/smartagentreceiver/README.md

```diff
@@ -20,17 +20,20 @@ should be used.
 1. Monitors with [dimension property and tag update
 functionality](https://dev.splunk.com/observability/docs/datamodel#Creating-or-updating-custom-properties-and-tags)
 allow an associated `dimensionClients` field that references the name of the SignalFx exporter you are using in your
-pipeline.  If you do not specify any exporters via this field, the receiver will attempt to use the associated
-pipeline.  If the next element of the pipeline isn't compatible with dimension update behavior, if a lone SignalFx
+pipeline.  These monitors include `ecs-metadata`, `heroku-metadata`, `kubernetes-cluster`, `openshift-cluster`, `postgresql`,
+and `sql`.
+If you do not specify any exporters via this field, the receiver will attempt to use the associated
+pipeline.  If the next element of the pipeline isn't compatible with dimension update behavior and if a lone SignalFx
 exporter was configured for your deployment, it will be selected.  If no dimension update behavior is desired,
 you can specify the empty array `[]` to disable.
 1. Monitors with [event-sending
-functionality](https://dev.splunk.com/observability/docs/datamodel/ingest#Send-custom-events) should be made members of
+functionality](https://dev.splunk.com/observability/docs/datamodel/ingest#Send-custom-events) should also be made members of
 a `logs` pipeline that utilizes a [SignalFx
 exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/signalfxexporter/README.md)
 that will make the event submission requests.  It's recommended, and in the case of the Processlist monitor required,
 to use a Resource Detection to ensure that host identity and other useful information is made available as event
```

Comment:
Is there a link to information on resource detection? Should we include that for anyone who may be new to configuring these monitors?

Reply:
typo (missing "processor"), and added a link.